docs/specs/git.yml

```diff
@@ -754,3 +754,60 @@ outputs:
       "original_content_git_url": "https://github.com/user-assigned/original/blob/original/index.md",
       "gitcommit": "https://github.com/testowner/testrepo/blob/450790f8a588ed8d96f4cb931c744d3855b7a809/index.md",
     }
+---
+# Put contributor list to .publish.json (Expose Contributor Information)
+repos:
+  https://github.com/expose-contributor/testrepo#master-file-cache:
+    - author: Charlie
+      email: charlie@contoso.com
+      files:
+        docs/index.md: |
+          ---
+          author: charlie
+          ---
+          Alice creates this. Charlie updates this. Bob updates this. Charlie becomes author.
+    - author: Bob 
```

Comment:
can we add a case that author is not one of the contributors?

Reply:
Added.